src/main/java/com/redhat/cloud/policies/app/health/ProcSelfStatusExporter.java

```diff
@@ -41,27 +42,45 @@
 @ApplicationScoped
 public class ProcSelfStatusExporter {
 
+    private final Logger log = Logger.getLogger(this.getClass().getSimpleName());
```

Comment:
`Logger` is no longer used in our apps. `io.quarkus.logging.Log` that was removed in this PR is the right way to go now.